cmd/thanos/query.go

```diff
@@ -479,3 +366,100 @@ func firstDuplicate(ss []string) string {
 
 	return ""
 }
+
+type queryConfig struct {
+	http httpConfig
+	//grpc configs
```

Comment:
do we need this comment? (:

Reply:
need, no :) however all the config in the commented block is related to grpc so was just highlighting that, happy to remove if you want.

Reply:
removed